packages/aztec.js/src/signer/index.js

```diff
@@ -62,7 +64,43 @@ signer.signNote = (verifyingContract, noteHash, spender, privateKey) => {
         status,
     };
     const { signature } = signer.signTypedData(domain, schema, message, privateKey);
-    return signature[0] + signature[1].slice(2) + signature[2].slice(2);
+    const r = signature[1];
+    const s = signature[2].slice(2);
+    const v = signature[0].slice(-2);
+    return r + s + v;
```

Comment:
Lets just have one function that returns the signature structure that is compatible with metamask (r + s + v).

Reply:
Ignore previous comments

Reply:
Good idea, done